app/GlucoseLink/CBPeripheral+UUIDString.m

```diff
@@ -10,8 +10,8 @@
 
 @implementation CBPeripheral (UUIDString)
 
-- (NSString*) UUIDString {
-  return (__bridge_transfer NSString *)CFUUIDCreateString(NULL, self.UUID);
+- (NSString *) UUIDString {
+  return self.identifier.UUIDString;
```

Comment:
You could kill off this category and update the clients to use this impl inline since it's now quite straightforward. I'd be happy to make that change too.